pep/app/pods/components/page/sidebar/widgets/template.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="p-3 d-flex">
+    <a href="" {{on "click" this.toggleAll}}>
+        {{t (if this.showCloseAll 'relatedInfo.batchClose' 'relatedInfo.batchOpen')}}
+    </a>
```

Comment:
This should be a <button> to be more semantically correct (and removes the need for the `event.preventDefault()` in the action), you can keep the link-like appearance by doing 
```hbs
<Button @type="link" class="p-0" .../>
```
Couple of other more design-related things i noticed here;
- The toggle button looks odd just hanging out at the top, i think now that we have a parent action control there, we should add back the "Related Info" h5 heading I had previously removed, and align the toggle to the right of the heading, to anchor it a bit better.
- The cards going right to the edges of the sidebar makes them look less "card-like", I think at least a little padding around everything (`p-2` seems like a decent amount) cleans that up a bit
- I am still pretty strongly against the action-icon-in-card-header pattern here 😛   I adjusted the html in devtools to show my suggested pattern in the screenshot below to visualize it better. I think thats the better way to go in this case for a bunch of reasons, but willing to bring it to a broader discussion if you want.

Current layout (for reference):
![current](https://user-images.githubusercontent.com/611458/91327522-a1dc4e80-e793-11ea-8213-d1c89487476e.png)

Adjusted:
![adjusted](https://user-images.githubusercontent.com/611458/91327535-a56fd580-e793-11ea-8dc6-d3af689374f1.png)